Survey/Sources/Supports/Extensions/Foundation/Constants+.swift

```diff
@@ -0,0 +1,31 @@
+//  swiftlint:disable:this file_name
+//  Constants+.swift
+//  Survey
+//
+//  Created by Khanh on 01/08/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Foundation
+
+extension Constants.API {
+
+    static let apiPathVersion = "/api/v1/"
+
+    static let baseUrl = Environment.based(
+        staging: "https://nimble-survey-web-staging.herokuapp.com" + apiPathVersion,
+        production: "https://survey-api.nimblehq.co" + apiPathVersion
+    )
+}
+
+// swiftlint:disable force_try
+extension Constants.Keys {
+
+    static var clientId: String {
+        try! Configuration.value(for: "CLIENT_ID")
```

Comment:
Try not to use force unwrap, when the app grows and we get crash, it will be harder to pinpoint where it went wrong.
```suggestion
        try? Configuration.value(for: "CLIENT_ID") ?? ""
```